zrml/prediction-markets/src/lib.rs

```diff
@@ -469,7 +469,7 @@ mod pallet {
             metadata: MultiHash,
             creation: MarketCreation,
             market_type: MarketType,
-            mdm: MarketDisputeMechanism<T::AccountId>,
+            dispute_mechanism: MarketDisputeMechanism<T::AccountId>,
```

Comment:
Because we changed the name of the extrinsic parameter here, we should probably tell the front-end about that change in the [changelog for devs](https://github.com/zeitgeistpm/zeitgeist/blob/main/docs/changelog_for_devs.md). https://github.com/zeitgeistpm/ui/search?q=mdm.